pkg/controller/perconaservermongodb/secrets.go

```diff
@@ -129,6 +135,13 @@ func (r *ReconcilePerconaServerMongoDB) reconcileUsersSecret(ctx context.Context
 	if err != nil {
 		return fmt.Errorf("create admin users pass: %v", err)
 	}
+	if cr.CompareVersion("1.13.0") >= 0 {
+		data["MONGODB_PERCONA_ADMIN_USER"] = []byte(rolePerconaAdmin)
+		data["MONGODB_PERCONA_ADMIN_PASSWORD"], err = secret.GeneratePassword()
+	}
```

Comment:
Should this user will be added to dbaas?

Reply:
we need to inform dbaas team that it was added and also if the password for this user is not set in secret it should not be empty in DB